Azure-Samples/azure-search-openai-javascript
# fix: changing developer settings sends the correct overrides

## Purpose



It looks like the overrides being done by developer settings were not being passed into the search call. This PR fixes that. 

## Does this introduce a breaking change?



```
[ ] Yes
[X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?



```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

- Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

- Test the code


```
Use developer settings and check if the value is being set correctly when sent to the search.
```

## What to Check

Verify that the following are valid

- ...

## Other Information
